torch/_dynamo/variables/nn_module.py

```diff
@@ -321,38 +324,46 @@ def call_function(
 
                 from .builder import wrap_fx_proxy
 
-                return wrap_fx_proxy(
-                    tx=tx,
-                    proxy=tx.output.create_proxy(
-                        "call_module",
-                        self.module_key,
-                        *proxy_args_kwargs(args, kwargs),
-                    ),
-                )
+                # TorchDynamo purposely graph breaks on RNN, GRU, LSTMs, don't try to make_fx and
+                # inline into these modules
+                if not config.use_single_step_graph or isinstance(
+                    mod, (torch.nn.RNN, torch.nn.GRU, torch.nn.LSTM)
```

Comment:
I was expecting to see an RNN test in the removed code here but I don't see it. So I guess you are duplicating this with somewhere else? Where is the somewhere else?

Reply:
Oh, the disjunction says that, when use_single_step_graph is True, it cannot handle RNN/GRU/LSTM, so just insert it as a call_module.

But it's pretty weird that you put it in as a call_module, as opposed to just... graph breaking? Like, wouldn't that be more straightforward? Then you could hoist this condition below.